css-view-transitions-2/Overview.bs

```diff
@@ -992,10 +992,10 @@ The [=captured element=] struct should contain these fields, in addition to the
 
 	1. If |oldDocument|'s [=Document/origin=] is not [=same origin=] as |newDocument|'s [=Document/origin=], then return false.
 
-	1. If |newDocument| [=was created via cross-origin redirects=], then return false.
-
 	1. If |navigationType| is {{NavigationType/traverse}}, then return true.
```

Comment:
If it's a non-BFCache traverse navigation, then the cross-origin restriction should still apply. It's no different from a push navigation then.

That said, we might have been overly conservative limiting navigations with cross-origin redirects. I don't quite remember the rationale other than "there could be an information leak and it's a rare use-case". So also happy to re-open that conversation if you'd like.

Reply:
Does the HTML spec's wording for the page swap event work here? https://html.spec.whatwg.org/multipage/browsing-the-web.html#updating-the-traversable:latest-entry

Reply:
Yes, I was thinking the same. Amended the PR